### PR TITLE
Quartz church: TextInput and TextArea components

### PR DESCRIPTION
**Pull request by whimsicallyson**
## Links
* quartz-church.glitch.me
* https://app.zenhub.com/workspaces/community-5d07d1359dee6059a9688176/issues/fogcreek/glitch-community-work/374
* Specs/doc links

## Changes:
* replaced internal textInput and textAreas with shared-components versions

## How To Test:
* Click around, try changing names of things, make sure you get errors when expected

## Things left to do before deploying:
- Also have an update to shared-components out to add focus styling (will need to bump version number to get that here)

### CODE REVIEW

**Comment by sarahzinger** [comment]
hello! i'm happy to review, but i see there are some merge conflicts, you mind resolving those and then ping me and I can take a look for ya?

**Comment by whimsicallyson** [comment]
@sarahzinger merged!

**Comment by whimsicallyson** [comment]
@sarahzinger updated, and this time circleci is happy as well :)

**Comment by sarahzinger** [comment]
@whimsicallyson is it supposed to match what's on prod or are there desired style changes?

I see for example on the homepage the search input looks different (not worse just different, want to make sure this is a desired change):
Screen Shot 2019-09-13 at 1 43 45 PM

**Comment by whimsicallyson** [comment]
@sarahzinger @ttseng updated search box styling form your inputs!

**Comment by ttseng** [comment]
thanks for adding the border!  can you also add a border radius?
`border-radius: 3px;`

image

**Comment by modernserf** [comment]
@ttseng should the border radius apply to all text inputs of that style (i.e. in the editor?)

**Comment by ttseng** [comment]
yes, that border radius is currently being applied to the main search in the editor and should be carried over once it uses shared components

image

**Comment by whimsicallyson** [comment]
This PR has been open for a while and I'm honestly not sure what its status is anymore. Do we need updates to shared-components before it can be merged? I don't *think* I need to update anything in this remix, but its possible...if so, what?

File: src/components/fields/collection-name-input.js [hunk 1]
```diff
@@ -9,6 +9,7 @@ const CollectionNameInput = ({ name, onChange }) => (
     value={name}
     onChange={onChange}
     placeholder="Name your collection"
+    style={{ fontWeight: 'bold' }}
```
Comment (sarahzinger) [inline src/components/fields/collection-name-input.js]: i think you may have missed a label here, when I go to a collection page I get an error in the console about a missing label

Comment (whimsicallyson) [reply src/components/fields/collection-name-input.js]: needed `label` instead of `labeltext` 👍

**Pull request merged**